server/handlers/genomic_annotations.go

```diff
@@ -143,7 +144,7 @@ func annotationExists(annotationName string) (exists bool, err error) {
 	return
 }
 
-func buildGetValuesQuery(params genomic_annotations.GetValuesParams) (string, error) {
+func buildGetValuesQuery(params genomic_annotations.GetValuesParams, query_test_data bool) (string, error) {
```

Comment:
Shouldn't `query_test_data` be camel case?

Reply:
Agreed